contrib/libvirt/files/libvirtd

```diff
@@ -0,0 +1,7 @@
+# libvirtd daemon service
+
+type       = process
+command    = /usr/bin/libvirtd
+before     = login.target
+depends-on = local.target
```

Comment:
are the dependencies for these correct? a quick glance at the (hilariously big) systemd files shows that at least libvirtd depends on virtlogd

no idea for the rest but at least that is wrong

Reply:
oh, lol, i actually couldn't find the systemd services, so i never checked :skull::laughing:

Reply:
step 1: install libvirt on any distro step 2: open a text editor

Reply:
All of my machines run chimera. Lmao